langgraph_bot/nodes/tnode/generate_image_node.py

```
@@ -0,0 +1,39 @@
+from langgraph_bot.agentschema.stateschema import State
+from langgraph_bot.utils.prompts import IMAGE_GENERATION_PROMPT
+
+
+load_dotenv()
+BASE_URL = "https://gen.pollinations.ai"
+
+
+def generate_image(state: State):
+    try:
+        secret_key = os.getenv("POLLINATIONS_SECRET_KEY")
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)